.gitignore

```diff
@@ -38,3 +38,16 @@ h2/db.mv.db
 src/main/react/tasklist/.vs
 src/main/react/tasklist/node_modules
 src/main/react/tasklist/build
+src/main/resources/.DS_Store
+
+workspace/organizations/*
```

Comment:
I would not ignore workspace orgnizations, forms and mails as the goal of this structure is to easily share the "low code" assets between developers